src/scripts/Modal.js

```diff
@@ -83,7 +83,7 @@ class Modal extends Component {
       if (this.props.isError) {
         className = classnames(className, 'slds-p-around--medium');
       }
-      return React.cloneElement(comp, { onClose: this.hide.bind(this), className });
```

Comment:
This was left over from copy-pasting, we still have and need the onClose but only on `ModalHeader` and not on `ModalContent`